airgun/views/computeresource.py

```diff
@@ -68,9 +68,8 @@ class http_proxy(View):
 
     @provider_content.register('Google')
     class GCEProviderForm(View):
-        google_project_id = TextInput(id='compute_resource_project')
-        client_email = TextInput(id='compute_resource_email')
-        certificate_path = TextInput(id='compute_resource_key_path')
+        json_key = TextInput(id='gce_json')
+        upload_button = Text("//input[contains(@id,'password_json']")
```

Comment:
Isn't this supposed to be a button?

Reply:
Yeah, we can go by this as well upload_button = Button("Choose File") @shweta83 thoughts?

Reply:
I don't think it would work. It is not a button type.